anna.m/lecture 1/site.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+
+    <head>
```

Comment:
Take content out of the head :) In head you inly have meta, title, style, scripts, links and that's it :) 
All content should be in body.
Also please use VS Code formatting because you have 4 spaces indentation when you should have 2 spaces.
Please check out how to automatically format files using VS Code, also if it still 4 spaces, make sure you have .editorconfig integration switched on.